src/main/scala/streaming/StreamMentionReplyier.scala

```diff
@@ -0,0 +1,23 @@
+package streaming
+
+import com.danielasfregola.twitter4s.{TwitterRestClient, TwitterStreamingClient}
+import com.danielasfregola.twitter4s.entities.Tweet
+import scala.concurrent.ExecutionContext.Implicits.global
+
+object StreamMentionReplyier extends App {
+
+  // TODO - Make sure to define your consumer and access tokens with write permissions!
+  val streamingClient = TwitterStreamingClient()
+  val restClient = TwitterRestClient()
+
+  def favoriteIfHashtag(tweet: Tweet) = tweet.entities match {
+    case None => ()
```

Comment:
`case None => ()` is not really elegant, can you please use map instead? It's going to be similar to what I have done in my commit for this PR